context/app/static/js/components/detailPage/StatusIcon/style.ts

```diff
@@ -1,15 +1,27 @@
 import { styled } from '@mui/material/styles';
-import LensIcon from '@mui/icons-material/LensRounded';
+import { InfoRounded, ErrorRounded, WarningRounded, CheckCircleRounded } from '@mui/icons-material';
+import { SvgIconProps } from '@mui/material/SvgIcon';
 
-interface ColoredStatusIconProps {
-  $iconColor: 'info' | 'success' | 'warning' | 'error';
+export interface ColoredStatusIconProps extends SvgIconProps {
+  $iconStatus: 'info' | 'success' | 'warning' | 'error';
 }
 
-const ColoredStatusIcon = styled(LensIcon)<ColoredStatusIconProps>(({ theme, $iconColor }) => ({
-  color: theme.palette[$iconColor].main,
-  fontSize: 16,
-  marginRight: 3,
-  alignSelf: 'center',
-}));
+export const iconSymbolStatusMap: {
+  [key in ColoredStatusIconProps['$iconStatus']]: React.ComponentType<SvgIconProps>;
+} = {
+  info: InfoRounded,
+  success: CheckCircleRounded,
+  warning: WarningRounded,
+  error: ErrorRounded,
+};
 
-export { ColoredStatusIcon };
+export const getStyledIcon = (
+  IconComponent: React.ComponentType<SvgIconProps>,
+  $iconStatus: ColoredStatusIconProps['$iconStatus'],
+) =>
+  styled(IconComponent)(({ theme }) => ({
+    color: theme.palette[$iconStatus].main,
+    fontSize: 16,
```

Comment:
Would this approach work? I see that it's inherited from the previous styles, but figure it's worth trying a conversion to rems here to match most of the other styles:

```suggestion
    fontSize: "1rem"
```